operator/pkg/controller-runtime/cell.go

```diff
@@ -45,10 +46,14 @@ func newScheme() (*runtime.Scheme, error) {
 	return scheme, nil
 }
 
-func newManager(lc hive.Lifecycle, logger logrus.FieldLogger, jobRegistry job.Registry, scope cell.Scope, scheme *runtime.Scheme) (ctrlRuntime.Manager, error) {
+func newManager(lc hive.Lifecycle, logger logrus.FieldLogger, jobRegistry job.Registry, scope cell.Scope, k8sClient client.Clientset, scheme *runtime.Scheme) (ctrlRuntime.Manager, error) {
```

Comment:
(nit) with this many parameters, I think a `cell.In` struct reads more nicely

Reply:
thanks, done